src/components/sortableGridList/sortableGridList.api.json

```diff
@@ -31,7 +31,7 @@
     {
       "name": "orderItemsByIndex",
       "type": "boolean",
-      "description": "Wether to reorder the items by index instead of by swapping locations."
+      "description": "Wether to reorder the items by index instead of by swapping locations.\nItems will move to the new index by pushing other items ahead or aback instead of swapping places with the item at the new index."
```

Comment:
Not sure `\n` works here

Reply:
Were? We can test it :)

Reply:
https://wix-private.github.io/wix-react-native-ui-lib/docs/components/incubator/Dialog#testid-1
https://github.com/wix/react-native-ui-lib/blob/a8b433ccf1d8cd659c4b7c833e4ded4bb9618033/src/incubator/Dialog/dialog.api.json#L34-L38

Reply:
Seems to work, hopefully the `space` is not required